pulp_puppet_plugins/pulp_puppet/plugins/distributors/installdistributor.py

```diff
@@ -96,6 +95,8 @@ def validate_config(self, repo, config, config_conduit):
         :rtype: tuple of length two. Either (False, str) or (True, None)
         """
         path = config.get(constants.CONFIG_INSTALL_PATH)
+        if not path:
```

Comment:
It looks like path is already being validated on line 100. 
It definitely makes sense for the puppet install distributor to check that a install_path is provided so maybe just update the return value on line 102